src/pages/index.js

```diff
@@ -192,13 +192,13 @@ function handleEditAvatarFormSubmit(value) {
       console.error("Avatar update error:", err);
     })
     .finally(() => {
-      avatarModal.setLoadingState(true);
+      avatarModal.setLoadingState(false);
     });
 }
 
 function handleDeleteModal(cardId, card) {
   deleteCardPopup.setFormSubmitHandler(() => {
-    deleteCardPopup.setDeleteState(true);
+    deleteCardPopup.setDeleteState(false);
     api
       .deleteCard(cardId)
       .then(() => {
```

Comment:
You shouldn't have changed `handleDeleteModal`. Everything was right there with the `Deleting...`